portfolio/src/main/webapp/script.js

```diff
@@ -53,8 +53,9 @@ function displayMessages() {
     console.log(messages);
     const messageContainer = document.getElementById('messages-container');
     messageContainer.innerText = '';
-    for (const message of messages) {
-      messageContainer.innerText += message + '\n';
+    for (const username in messages) {
+      messageContainer.innerText += username + ': ' + messages[username] + '\n\n';
```

Comment:
Do the \n\n have the desire effect?  Since this is HTML?

Reply:
I believe it did, but I'll render again and check.